module/data/actor/ability.mjs

```diff
@@ -0,0 +1,26 @@
+import { FormulaField } from "../fields.mjs";
+
+/**
+ * An embedded data structure for actor ability scores.
+ * @see ActorData5e
+ *
+ * @property {number} value          Ability score.
+ * @property {number} proficient     Proficiency value for saves.
+ * @property {object} bonuses        Bonuses that modify ability checks and saves.
+ * @property {string} bonuses.check  Numeric or dice bonus to ability checks.
+ * @property {string} bonuses.save   Numeric or dice bonus to ability saving throws.
+ */
+export default class AbilityData extends foundry.abstract.DataModel {
```

Comment:
I would prefer for something relatively small like each ability to instead be a `SchemaField` instead of a full embedded `DataModel`. Using an embedded data model is more heavyweight and unless we need that embedded data model to contain a lot of specialized logic we would be better served using the more lightweight tool. 

You've used this approach elsewhere like with `makeResourceField` which returns a `SchemaField` for each resource pool. Can we do the same here?

Reply:
It could be useful, particularly for ability scores and skills, to be able to move preparation logic for preparing a single ability score/skill inside these data model classes. Those two have reasonably involved preparation logic.

Reply:
Yeah, moving the preparation logic for abilities and skills here is what I was thinking.